eZ/Publish/Core/Search/Common/FieldNameResolver.php

```diff
@@ -101,17 +92,17 @@ protected function getFieldMap()
                         continue;
                     }
 
-                    $this->fieldTypes[$contentType->identifier][$fieldDefinition->identifier] =
+                    $fieldTypes[$contentType->identifier][$fieldDefinition->identifier] =
                         $fieldDefinition->fieldType;
                 }
             }
         }
 
-        return $this->fieldTypes;
+        return $fieldTypes;
```

Comment:
As you can seem ut is generated for every run, this was because of issues with caching this locally. @pspanja did you find another place to cache this calculations on the misc field type related handlers?